src/client/java/daos-java/src/main/native/io_daos_DaosClient.c

```diff
@@ -225,7 +227,10 @@ Java_io_daos_DaosClient_daosGetContAttrs(JNIEnv *env,
 	}
 
 	memcpy(&coh, &contHandle, sizeof(coh));
-	rc = daos_cont_get_attr(coh, n, names, values, sizes, NULL);
+	rc = daos_cont_get_attr(coh, n,
+				(const char * const *)names,
+	 			(void * const *)values,
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs